products/casp/main.pm

```diff
@@ -32,26 +32,42 @@ my $distri = testapi::get_var("CASEDIR") . '/lib/susedistribution.pm';
 require $distri;
```

Comment:
Just a thought; seems a bit strange to be doing this, maybe a bit dangerous as well